src/tools/debugger/session/Breakpoints.java

```diff
@@ -117,6 +117,7 @@ private Breakpoint saveTruffleBasedBreakpoints(final SectionBreakpoint bId) {
     FullSourceCoordinate coord = bId.getCoordinate();
     BreakpointEnabling<T> existingBP = breakpoints.get(coord);
     if (existingBP == null) {
+      WebDebugger.log("SetSectionBreakpoint: " + bId);
```

Comment:
in the`saveTruffleBasedBreakpoints` method is also used this log, should I remove them both?